config/routes.rb

```diff
@@ -48,5 +48,5 @@
   get 'acknowledge', to: 'messages#acknowledge'
 
   # Route everything else to CKAN
-  get '*path' => redirect('http://ckan.publishing.service.gov.uk/')
+  get '*path' => redirect(Rails.configuration.ckan_redirection_url)
```

Comment:
Should this be wrapped in a `if` block?  I'm wondering what users would get redirected to if the config value isn't present (e.g. locally in development)?

Reply:
Yes, it makes sense, I'm going to fix this

Reply:
Done